astropy/io/fits/tests/test_hdulist.py

```diff
@@ -692,7 +692,7 @@ def test_fromstring(filename):
         for filename in glob.glob(os.path.join(self.data_dir, '*.fits')):
             if sys.platform == 'win32' and filename == 'zerowidth.fits':
                 # Running this test on this file causes a crash in some
-                # versions of Numpy on Windows.  See PyFITS ticket
+                # versions of Numpy on Windows.  See ticket:
```

Comment:
If we really want to get rid of PyFITS reference, we should not reference to old PyFITS ticket (it also might disappear in the future). Better to open a new issue for this in Astropy and then link to that new issue.

Reply:
Yeah I didn't touch those. I thought they might stay around...

Reply:
Those Trac sites are at the mercy of STScI ITSD. No promises!

Reply:
There are lots of these links, should we really open new issues for these?

Reply:
Oh... Sure. If you are not ready to tackle the Trac links now, let's open a new issue for it.

Reply:
Unfortunately, I can't do anything with this PR right now 😄 

Will open a new issue for it.